flow-typed/team_types.js

```diff
@@ -0,0 +1,8 @@
+// @flow
+
+export type Team$TeamMemeber = {
+  userName: string,
```

Comment:
Maan, you're gonna hate me. What's up with `camelCase` and `snake_case` notations here? 😈

Reply:
Did you fix this?